hash/apple2_flop_misc.xml

```diff
@@ -49,6 +49,25 @@ Generated by SLIST 0.2.1
 		</part>
 	</software>
 
+	<software name="a2diag31">
+		<description>Apple II Diagnostic Disk (version 3.1)(800K 3.5")</description>
+		<year>1991</year>
+		<publisher>Apple Computer</publisher>
+		<info name="usage" value="Requires an Apple IIgs, IIc+, //c, //e with a compatible drive controller card." />
+		<sharedfeat name="compatibility" value="A2E,A2EE,A2C,A2GS" />
+		<!-- This disk contains 4 versions of the diagnostic tools:
+		    Apple IIgs Diagnostic, V3.1
+		    Apple IIc Plus Diagnostic, V2.0
+		    Apple IIc Diagnostic, V3.0
+		    Apple IIe Diagnostic, V3.0
+		-->
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="819200">
+				<rom name="apple ii diagnostic v3_1.po" size="819200" crc="50d3243d" sha1="23881d0096d428f2ea8568c854dc25f6b66ff051" />
+			</dataarea>
+		</part>
+	</software>
+
```

Comment:
There’s no dump credit for this – where did it come from?

Reply:
> There’s no dump credit for this – where did it come from?

It has been a while since I had it.  I see it is available from at least two sites without any credits given on either one. 

Do all submitted dumps need a credit?  If so, I will remove it since I mostly just added it to support testing the PR.

Reply:
Credit yourself or anonymous.

Reply:
I have credited anonymous in the main PR description just now since I did not create the dump myself.  Is it just in the PR form that I do this?